ops/src/main/scala/ammonite/ops/Shellout.scala

```diff
@@ -81,6 +84,9 @@ object Shellout{
           ))
         }
       }
+      // if we did not read anything sleep briefly to avoid spinning
```

Comment:
Only sleep if we have not seen input during this iteration.  This means that for fast processes we do not add unnecessary pauses.